### PR TITLE
Storyblok backup

### PR DESCRIPTION
**Pull request by sophieheb**

### CODE REVIEW

File: db/migrate/20191002180118_create_backups.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateBackups < ActiveRecord::Migration[5.2]
+  def change
+    create_table :backups, id: :uuid do |t|
+      t.references :user, foreign_key: true, type: 'uuid'
```
Comment (jits) [inline db/migrate/20191002180118_create_backups.rb]: Doesn't look this field is being used or is needed?

File: db/schema.rb [hunk 2]
```diff
@@ -10,12 +10,29 @@
...
 
+  create_table "backups", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.uuid "user_id"
+    t.string "key"
+    t.jsonb "data"
+    t.index ["key"], name: "index_backups_on_key", unique: true
+    t.index ["user_id"], name: "index_backups_on_user_id"
+  end
+
+  create_table "storyblok_backups", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
```
Comment (jits) [inline db/schema.rb]: This looks like it shouldn't be here?

File: db/migrate/20191002180118_create_backups.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+      t.string :key
```
Comment (jits) [inline db/migrate/20191002180118_create_backups.rb]:
```suggestion
      t.string :key, null: false
```

File: db/migrate/20191002180118_create_backups.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+      t.jsonb :data
```
Comment (jits) [inline db/migrate/20191002180118_create_backups.rb]:
```suggestion
      t.jsonb :data, null: false, default: {}
```

File: app/models/backup.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+class Backup < ApplicationRecord
+end
```
[the remark this answers is not in the document]
Comment (sophieheb) [reply app/models/backup.rb]: Done.

File: db/schema.rb (hunk 2 above)
Comment (sophieheb) [reply db/schema.rb]: Yep, old version snuck in 🤦‍♀

Comment (sophieheb) [reply db/schema.rb]: It's gone now!

File: db/migrate/20191002180118_create_backups.rb (hunk 1 above)
Comment (sophieheb) [reply db/migrate/20191002180118_create_backups.rb]: Deleted.

**Pull request merged**